doc/reference/reference_sql/sql-features.rst

```diff
@@ -14,443 +13,1032 @@ else we will mark it "Fail".
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests 
+        *   -   F471
+            -   Scalar subquery values
+            -   ``SELECT s1 FROM t WHERE s1 = (SELECT count(*) FROM t);``   
+            -   Okay.
+        *   -   F481   
+            -   Expanded NULL зredicate
+            -   ``SELECT * FROM t WHERE row(s1,s1) IS NOT NULL;``   
+            -   Fail. Syntax error.   
+        *   -   F812   
+            -   Basic flagging   
+            -
+            -   Fail. Tarantool doesn't support any flagging.   
+        
+   
+S011, Distinct types
+--------------------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests 
+        *   -   S011   
+            -   Distinct types
+            -   ``CREATE TYPE x AS FLOAT;``   
+            -   Fail. Tarantool doesn't support distinct types.   
+  
+   
+T321, Basic SQL-invoked routines
+--------------------------------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests 
+        *   -   T321-01   
+            -   User-defined functions with no overloading
+            -   ``CREATE FUNCTION f() RETURNS INT RETURN 5;``   
+            -   Fail. User-defined functions for SQL are created in   
+                :ref:`Lua <sql_calling_lua>` with a different syntax.   
+        *   -   T321-02   
+            -   User-defined procedures with no overloading
+            -   ``CREATE PROCEDURE p() BEGIN END;``   
+            -   Fail. User-defined functions for SQL are created in   
+                :ref:`Lua <sql_calling_lua>` with a different syntax.   
+        *   -   T321-03
+            -   Function invocation
+            -   ``SELECT f(1) FROM t;``   
+            -   Okay. Tarantool can invoke Lua user-defined functions.
+        *   -   T321-04   
+            -   CALL statement
+            -   ``CALL p();``   
+            -   Fail. Tarantool doesn't support CALL statements.   
+        *   -   T321-05   
+            -   RETURN statement
+            -   ``CREATE FUNCTION f() RETURNS INT RETURN 5;``   
+            -   Fail. Tarantool doesn't support RETURN statements.   
+
+
+T*, Other
+---------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests 
+        *   -   T631   
+            -   IN predicate with one list element
+            -   ``SELECT * FROM t WHERE 1 IN (1);``   
+            -   Okay.   
+
+
+Total number of items marked "Fail": 67
+
+Total number of items marked "Okay": 79
```

Comment:
Please add trailing newline

Reply:
There is a trailing newline, its number is 1045.